src/main/java/org/kohsuke/github/GitHubRequest.java

```diff
@@ -75,7 +75,7 @@ private GitHubRequest(@Nonnull List<Entry> args,
 
     /**
      * Create a new {@link Builder}.
-     * 
+     *
```

Comment:
Wow, the built in formatter doesn't protect against this?  Ouch.

Reply:
Ugh. My editor is _supposed_ to be configured to remove trailing whitespace. Will clean this stuff up.

Reply:
Oh! I see. My editor _did_ remove it. Hah, now I understand.

Reply:
Yeah, it's the autoformatter in the build that failed to catch these earlier. 😢